### ISSUE TITLE
Automatic Image Generation Only Works for People with Write Access

### ISSUE THREAD

**Issue by jrg94** [TheRenegadeCoder/sample-programs-website#377]
See https://github.com/TheRenegadeCoder/sample-programs-website/pull/375

**Comment by jrg94** [comment]
I'll probably add the following to the action:

```
    # The name of the user that will be displayed as the author of the commit
    # Default: author of the commit that triggered the run
    author_name: Your Name

    # The email of the user that will be displayed as the author of the commit
    # Default: author of the commit that triggered the run
    author_email: mail@example.com
```

Not sure it will help, but I'll give it a shot!

**Comment by jrg94** [comment]
Alright, I toyed around with this quite a bit. I gave @stuin write access to the repo, and nothing changed. I also changed the author to myself, but I didn't have the patience to see if that worked. I suspect this is an issue with forks. We'll have to wait on the action author to get back to me. Otherwise, the image generation seems to work fine for me at least. See: https://github.com/TheRenegadeCoder/sample-programs-website/pull/381.
